docs/guides/lint.md

```diff
@@ -3,75 +3,237 @@ redirectFrom:
   - /docs/cli/configuration/lint/
 ---
 
-# Lint configuration
+## Introduction
 
 The `lint` configuration section is part of the [Redocly configuration file](../configuration/configuration-file.mdx).
-It is used by the `lint` and `bundle` commands to control various options.
+The `lint` and `bundle` commands use this section to control various options.
 
-Modify (or create) the `.redocly.yaml` file in the directory from which you are going to run the `lint` or `bundle` commands.
+The `lint` configuration section consists of several sub-sections (each section is described after this block):
 
-:::attention
+```yaml
+lint:
+  plugins:
+    - './local-plugin.js'
+  extends:
+    - recommended
+  resolve:
+    http:
+      headers:
+        - matches: https://api.example.com/v2/**
+          name: X-API-KEY
+          value: <direct value>
+          envVariable: <name of env variable to be used as value>
+        - matches: https://example.com/*/test.yaml
+          name: Authorization
+          value: <direct value>
+          envVariable: <name of env variable to be used as value>
+  preprocessors:
```

Comment:
issue: I think these (preprocessors and decorators) sections aren't allowed to be empty. I know you let them know they can omit the section... but really, they should omit the section.

Reply:
So, is it better to remove both sections from the example completely?

Reply:
Removed both sections from the example